ui/package.json

```diff
@@ -11,14 +11,12 @@
     "lint": "yarn lint:eslint",
     "lint:eslint": "eslint '**/*.{js,jsx}'",
     "lint-fix": "eslint --fix '**/*.{js,jsx}'",
-    "start": "react-scripts start",
+    "start": "yarn build:ses && react-scripts start",
     "test": "react-scripts test",
     "test:debug": "react-scripts --inspect-brk test --runInBand --no-cache",
-    "build": "yarn build:react",
+    "build": "yarn build:ses && yarn build:react",
     "build:ses": "cp ../node_modules/ses/dist/lockdown.umd.js public/",
-    "build:react": "yarn configure; yarn build:pre-patch; react-scripts build; yarn build:post-patch",
-    "build:pre-patch": "node ./ses-patch.mjs src/install-ses-lockdown.js",
-    "build:post-patch": "node ./ses-patch.mjs build/index.html",
```

Comment:
You might consider using a script instead of `cp`. Relative paths are more fragile than dependency resolution with a tool like `importMetaResolve`. The package manager is not obliged to put `ses` in this particular location.

Reply:
That makes sense. I'd like to just merge this change now, but that would be good to have, other dapps would likely use it too